examples/general/custom_routes/lib/pwa/offline_urls.g.dart

```diff
@@ -1,14 +1,46 @@
 /// URLs for offline cache.
 final List<String> offlineUrls = [
   './',
+  './.build.manifest',
+  './.packages',
+  './favicon.png',
   './main.dart.js',
-  './main.dart.js_1.part.js',
   './manifest.json',
-  './packages/browser/dart.js',
-  './packages/browser/interop.js',
+  './packages/\$sdk/_internal/ddc_sdk.sum',
+  './packages/\$sdk/_internal/strong.sum',
+  './packages/\$sdk/dev_compiler/amd/dart_sdk.js',
```

Comment:
I think these are not present in a prod release (`dart2js`). Although it is unlikely that these will hurt much (won't take up disk space), we should probably include only an example that has the final files cached offline.